plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -595,10 +614,21 @@ static gboolean
 fu_pxi_device_setup (FuDevice *device, GError **error)
 {
 	FuPxiDevice *self = FU_PXI_DEVICE (device);
+	g_autofree gchar *devid = NULL;
+	if (!fu_pxi_device_get_raw_info (self, &self->hid_raw_info ,error))
+		return FALSE;
 	if (!fu_pxi_device_fw_ota_init (self, error))
 		return FALSE;
 	if (!fu_pxi_device_fw_get_info (self, error))
 		return FALSE;
+
+	devid = g_strdup_printf ("HIDRAW\\VID_%04hX&PID_%04hX&BLE_DEV_%s",
+				self->hid_raw_info.vendor,
+				self->hid_raw_info.product,
+				fu_device_get_name (device));
+
+	fu_device_add_instance_id (device, devid);
+	fu_device_set_id (device, devid);
```

Comment:
Don't you mean `fu_device_set_logical_id()`?

Reply:
remove `fu_device_set_id `